src/main/java/net/apnic/rdap/rdap/RDAPError.java

```diff
@@ -0,0 +1,98 @@
+package net.apnic.rdap.rdap;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Class represent an error response in the RDAP protocol.
+ *
+ * @see https://tools.ietf.org/html/rfc7483
+ */
+public class RDAPError
+    extends RDAPObject
+{
+    private List<String> description = null;
```

Comment:
(suggestion) Do you ever expect to receive this from the client? If not there is an opportunity for immutability here - make all the fields final and ditch the setters.